kolena/workflow/visualization/_utils.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2021-2023 Kolena Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import io
+
+import numpy as np
+from PIL import Image
+
+from kolena.errors import InputValidationError
+
+
+def create_png(image: np.ndarray, mode: str) -> io.BytesIO:
```

Comment:
Should `mode` default to `RGB`?

Reply:
Yeah I thought about this but decided to not specify the default mode because for `SegmentationMask` use case, it will be a "L" (grayscale), and for `BitmapMask` use case, it will be a "RGBA".